### ISSUE TITLE
How to benchmark web application end points with specific rates, if possible?

### ISSUE THREAD

**Issue by ribubrar** [dotnet/crank#494]
Hi @sebastienros 
I am looking to benchmark different web application endpoints with specific rates, and those endpoints are invoking methods I am looking to benchmark.

Regarding the previous thread, [customizing requests per second](https://github.com/dotnet/crank/issues/489), it works perfectly well for benchmarking the whole application at different rates. 

I am wondering if I can achieve something like calling "/users" at rate 5, "/products" at rate 100, and "/customers" at rate 10?

```
[Route("users")]
public User GetUsers() { }
...

[Route("products")]
public User GetProducts() { }
...

[Route("customers")]
public User GetCustomers() { }
...
```

Thank you in advance!

**Comment by sebastienros** [comment]
Yes, the bombardier client has a `rate` variable that you can set in a scenario. So in your case you would have 3 scenarios, each targeting a different endpoint with a different variable like `rate: 5`

**Comment by ribubrar** [comment]
Thanks for that @sebastienros . Does that mean I would need to run the controller separately for each scenario? 

```
crank --config C:\Project\benchmarks.yml --scenario users --profile local --sql "..." --table CrankResult

crank --config C:\Project\benchmarks.yml --scenario products --profile local --sql "..." --table CrankResult

crank --config C:\Project\benchmarks.yml --scenario customers --profile local --sql "..." --table CrankResult
```

**Comment by sebastienros** [comment]
You can have multiple services in your scenario, can call them `load_users`, `load_customer`, `load_products` for instance, and the results would show 3 tables with different RPS. You wouldn't be able to distinguish the server-side metrics for each results though. If you care then you need 3 different runs on 3 different scenarios.
